src/commands/env.js

```diff
@@ -67,4 +67,25 @@ async function importEnv (params) {
   Logger.println('Environment variables have been set');
 };
 
-module.exports = { list, set, rm, importEnv };
+async function importEnvVars (params) {
+  const [envNames] = params.args;
+  const { alias } = params.options;
+
+  for (const envName of envNames) {
+    const nameIsValid = validateName(envName);
+    if (!nameIsValid) {
+      throw new Error(`Environment variable name ${envName} is invalid`);
+    }
+  }
+
+  const { org_id, app_id: appId } = await AppConfig.getAppData(alias).toPromise();
+
+  for (const envName of envNames) {
+    const value = process.env[envName] || '';
+    await application.updateEnvVar({ id: org_id, appId, envName }, { value }).then(sendToApi);
+  }
```

Comment:
I'd rather have one call with all variables at once.

Right now it's not really a problem because if you have 3 vars and if the 3rd one fails, you run the command again and since `updateEnvVar` is idempotent so it's no big deal.

Having one call will be a bit faster and will have just one opportunity for network failure.

Reply:
There's nothing in the API for that, though, ATM, and I'm not sure we want to add such an endpoint just for this, as it would be confusing VS the "override all the env" one.
We could fetch the whole env, edit it and then repush it but isn't that actually more overkill?

Reply:
There is ;-)

We have this:

```
PUT /organisations/{id}/applications/{appId}/env
PUT /self/applications/{appId}/env
```

It's used by the CLI and the console.

Reply:
OK, again, I just understood my mistake.